corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -260,10 +260,15 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
         self.geocoderItemCallback = function (item) {
             self.rawAnswer(item.place_name);
             self.editing = false;
+            var broadcastObj = Utils.getBroadcastObject(item);
             self.broadcastTopics.forEach(function (broadcastTopic) {
-                var broadcastObj = Utils.getBroadcastObject(item);
                 question.parentPubSub.notifySubscribers(broadcastObj, broadcastTopic);
             });
+            if ((!broadcastObj || _.isEmpty(broadcastObj)) && question.answer()) {
```

Comment:
I think this could be simplified to `_.isEmpty(broadcastObj)` because
* `_.isEmpty` should return false for any falsy value, so `!broadcastObj` isn't adding aything
* If `broadcastObj` is empty and the question is already set to `NO_ANSWER`, I'd think you'd want to execute this first block, not the second block (which would set the answer to a stringified version of `broadcastObj`). It's fine to re-set the value of `answer` even if it's already `NO_ANSWER` - subscribers won't get called unless the value changes.